fabric-1.19/build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+	// TODO: switch to a stable Loom release once 1.19 lands
+	id 'fabric-loom' version '0.12-SNAPSHOT'
```

Comment:
0.12-SNAPSHOT uses the latest version of 0.12 which means all futures patches (don't change the API at all)

Reply:
I really prefer to pin dependency versions to avoid the build breaking :)